src/main/java/uk/gov/hmcts/cmc/claimstore/documents/DummyBulkPrintService.java

```diff
@@ -15,7 +15,7 @@
     private static final Logger logger = LoggerFactory.getLogger(DummyBulkPrintService.class);
 
     @Override
-    public void print(Claim claim, List<Printable> documents) {
+    public void print(Claim claim, List<Printable> documents, String authorisation) {
         logger.info("No bulk print operation need to be performed as 'Bulk print url' is switched off.");
```

Comment:
Perhaps not for this PR, but we should consider amending this message to give more details, for the sake of local testing.